### DISCUSSION TITLE
Exploring and evaluating embedding models

### DISCUSSION THREAD

**Discussion by mrepetto94** [imartinez/privateGPT#1966] [Ideas]
I'm about to fine-tune the plain vanilla embedding model I'm using, but in order to do that, I would like a way to explore the current embedding space and evaluate the performances to keep track of the embedding model improvement. I'm well aware that this is not the scope of PrivateGPT, but I would like to hear from you about how you approached the problem. Are you using Qdrant Web UI? Or are you using something different?

And what are the frameworks you are using for evaluation?

**Comment by ph-m-m** [comment]
I’m currently working on a similar topic and would love to hear about the solution and frameworks you ended up using. Would you mind sharing your experience?
